server/svix-server/src/main.rs

```diff
@@ -62,9 +66,17 @@ async fn main() {
 
     tracing_subscriber::fmt::init();
 
-    if let Some(Commands::Migrate) = &args.command {
-        let _ = db::init_db_and_run_migrations(&cfg).await;
+    async fn run_migrations(cfg: &Configuration) {
```

Comment:
I only see one function `init_db_and_run_migrations` there that runs migrations. There is a second function that simply awaits `init_db_and_run_migrations`, but rust-analyzer flags it as inactive code. I can remove that, but unsure what else to do there.

Reply:
It's inactive because it depends on a configuration. It'll be active in a release build. You need to clean it up to make sure that `init_db` and `run_migrations` are NOT called (even in release builds) when you don't pass this flag...